components/ContentFlow.brs

```diff
@@ -228,6 +228,8 @@ sub onTruexAdDataReceived(event as object)
     m.adRenderer.action = tarInitAction
 
     ? "TRUE[X] >>> ContentFlow::onTruexAdDataReceived() - starting TruexAdRenderer..."
+    ' CTV-1745 - must stop the video player since Roku only allows one Video stream to buffer
+    m.videoPlayer.control = "stop"
```

Comment:
Do we not need any resume logic on this? What about the handling of ad skipping? Does the seeking logic work with the video player stopped?

Reply:
The resume/seeking logic is already implemented and seems to be working during my smoke test.

When the ad break begins [the stream position is saved.](https://github.com/socialvibe/truex-roku-google-ad-manager-reference-app/blob/master/components/ContentFlow.brs#L203)
When the engagement ends [the stream is played from that position](https://github.com/socialvibe/truex-roku-google-ad-manager-reference-app/blob/master/components/ContentFlow.brs#L355) plus the length of the ad break.

Reply:
ah, thanks for confirming nice - makes this change very simple